langchain4j-mistral-ai/src/test/java/dev/langchain4j/model/mistralai/MistralAiCompletionModelIT.java

```diff
@@ -0,0 +1,131 @@
+package dev.langchain4j.model.mistralai;
+
+import dev.langchain4j.model.language.LanguageModel;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.output.TokenUsage;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static dev.langchain4j.model.output.FinishReason.STOP;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class MistralAiCompletionModelIT {
+
+    LanguageModel codestral = MistralAiCompletionModel.withApiKey(System.getenv("MISTRAL_AI_API_KEY"));
+
+    @Test
+    void should_generate_code_completion_and_return_token_usage_and_finish_reason_stop() {
+        // Given
+        String codePrompt = "public static void main(String[] args) {";
+        // When
+        Response<String> response = codestral.generate(codePrompt);
+        // Then
+        System.out.println(
```

Comment:
this is not a proper test. what about having a prompt "public static void main" and expecting "String[] args" in the output?